app/view_models/auction_show_view_model.rb

```diff
@@ -172,7 +172,7 @@ def user_is_bidder?
 
   def html_description
     return '' if auction.description.blank?
-    markdown.render(auction.description).html_safe
+    MarkdownRender.new(auction.description).to_s.html_safe
```

Comment:
Good plan, but I wonder if we should make a method on the MarkdownRender class that combines the `to_s.html_safe`